user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -0,0 +1,281 @@
+
+## Navigation
+
+In order to have consistent navigation, we recommend applying the following order for navigation in a low-code app:
+
+1. Sidebar: Main navigation to the different pages defined in the app.
+
+   ![Navigation via sidebar](~/user-guide/images/LCA_Style_Guide_Navigation_Pages.png)
+
+   > [!NOTE]
+   > Use a side panel to show additional information for each item on a page.
+
+1. Header bar: Navigation within a page.
+
+   This can open up a hidden page, in which case it makes sense to change the title of each page so users know where they are.
+
+   ![Navigation via menubar](~/user-guide/images/LCA_Style_Guide_Navigation_Menu.gif)
+
+> [!IMPORTANT]
+> It is generally **not** advised to use buttons for navigation.
+
+## Panels
+
+- Panels in a low-code app should primarily be used to show additional information about an item selected on a page. This can for example be a DOM form or additional details/metadata that were not shown on the original page.
+
+- Panels can be shown on the left or the right of the screen or as a pop-up window. Try to always show a panel in the same location in an app, with the same width, etc. As a default, we recommend showing DOM forms in a pop-up window and showing additional information about a selected table row in a side panel on the right.
+
+  ![Form pop-up window](~/user-guide/images/LCA_Style_Guide_Form_Popup.gif)
+
+- Every panel should have a close button in the top-right corner. This can be in the menu bar or it can be a separate button.
+
+## Components
+
+### Titles
+
+- Titles are optional on pages, depending on whether users will need more context information about what they are looking at.
+
+- Avoid repeating the app name as a title on an app's pages.
+
+- Do not use a background color for titles.
+
+- Do not make the titles too big: the focus should be on the content of the page.
+
+- The best way to add a title to a page is as a web component. For example:
+
+  - HTML code:
+
+    ```html
+    <div style="height:100%;display:flex;flex-direction:column;justify-content:center">
+      <H1 style="color: rgb(20,33,42); font-family:'Segoe UI Light','Segoe UI Web Light','Segoe UI Web Regular','Segoe UI',Arial,sans-serif;margin:0;font-weight:100;font-size:24px">
+        <b>People</b>
+      </H1>
+      <p style="color: rgb(20,33,42,0.7); font-family:'Segoe UI Light','Segoe UI Web Light','Segoe UI Web Regular','Segoe UI','Segoe UI Symbol',HelveticaNeue-Light,'Helvetica Neue',Arial,sans-serif;margin:0;font-weight:600;font-size:16px">
+      An overview of all managed people
+      </p>
+    </div>
+    ```
+
+  - Result:
+
+    ![Good title](~/user-guide/images/LCA_Style_Guide_Title_Good.png)
+
+### Buttons
+
+[Buttons](xref:DashboardButton) can trigger different actions on a page or panel. Here are some things to keep in mind when you add buttons:
+
+- Use short labels.
+
+- Make sure buttons have a transparent background.
+
+   | 👍 Good | 👎 Bad |
```

Comment:
why is the example in between the bullet points. Maybe we can put it at the end.